compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -144,6 +145,9 @@
     private File iosResourceRulesPList;
     @Element(required = false)
     private File iosEntitlementsPList;
+    
+    @Element(required = false)
+    private Tools tools = new Tools();
```

Comment:
I believe that this has to be `null` by default to avoid `Builder.write(...)` from printing out an empty section when dumping a `Config` as XML.